lib/elixir/lib/enum.ex

```diff
@@ -2129,17 +2132,22 @@ defmodule Enum do
   end
 
   @doc """
-  Returns a subset list of the given enumerable, from `range.first` to `range.last` positions.
+  Returns a subset list of the given enumerable, from `index_range.first` (zero-based) to
```

Comment:
Backticks on "enumerable"? (same for the next function)

Reply:
done, and applied in a new commit to the rest of the module